tests/_support/formChildCommands.js

```diff
@@ -0,0 +1,181 @@
+const StringUtil = require("./utils/StringUtil");
+
+/**
+ * Select the input element of the form group with the given label
+ *
+ * NOTE: This function ignores case and removes extraneous whitespaces, yet
+ *       it performs a complete label match.
+ *
+ * This utility is used for selecting form elements by their visual name instead
+ * of using a more specific CSS selector.
+ *
+ * This detects DOM structures like:
+ *
+ * <div class="form-group">
+ *   <div ...
+ *     <label>Track Name</label>
+ *   </div>
+ *   <div ...
+ *     <input />    # .. or
+ *     <select />   # .. or
+ *     <textarea /> # .. or
+ *     <div class="form-control" />
+ *   </div>
+ * </div>
+ *
+ * @param {jQuery.Element} elements - The DOM scope to search within
+ * @param {String} label - The contents of the <label> to match
+ */
+Cypress.Commands.add("getFormGroupInputFor", { prevSubject: true }, function(
```

Comment:
Is this method `getFormGroupInputFor` repetition expected? because file `system-tests/_support/formChildCommands.js` also contain the same method `getFormGroupInputFor `